web_action_conditionable/readme/CREDITS.md

```diff
@@ -0,0 +1,2 @@
+The migration of this module from 15.0 to 17.0 was financially supported
```

Comment:
is this actually the case or a fluke of replacing version numbers?

Reply:
strictly speaking from 15.0 to 16.0 and then from 16.0 to 17.0, but yes, it is the case